server/protocols/webadmin/webadmin-mailbox/src/main/java/org/apache/james/webadmin/routes/UserQuotaRoutes.java

```diff
@@ -139,6 +151,88 @@ public void defineGetQuota() {
         }, jsonTransformer);
     }
 
+    @GET
+    @ApiOperation(
+        value = "Reading count and size at the same time",
+        notes = "If there is no limitation for count and/or size, the returned value will be -1"
```

Comment:
could we stop with -1 special value and use null or undefined instead ?

Reply:
That would affect behaviour, and break old endpoints, potentially OpenPaaS....

Reply:
It's a new endpoint, isn't it ? there's no backward compatibility issue

Reply:
That re-uses other DTOs and is similar to other endpoints.

By doing like this we will soon end up with inconsistent APIs. Is that really what we want?

I'd value consistency. If we change this here, we'd have to change it in every quota endpoints.

Reply:
I agree with replacing this special value by another one, but in another issue; as we will have to change all the quota endpoints.